src/libsync/account.cpp

```diff
@@ -526,10 +530,26 @@ void Account::setSslErrorHandler(AbstractSslErrorHandler *handler)
 
 void Account::setUrl(const QUrl &url)
 {
-    _url = url;
+    const QRegularExpression discoverPublicLinks(R"((http.://[^/]*).*/s/([^/]*))");
```

Comment:
would be nice to have plain `http://` links work too (e.g. for local dev)

```suggestion
    const QRegularExpression discoverPublicLinks(R"((https?://[^/]*).*/s/([^/]*))");
```

Reply:
done
we will accept `http` and `https` protocols